sdk/servicebus/service-bus/.scripts/buildBrowserTestResources.js

```diff
@@ -0,0 +1,352 @@
+console.log("Creating browser resources...");
+
+const msRestNodeAuth = require("@azure/ms-rest-nodeauth");
+
+const ServiceBusManagementClient = require("@azure/arm-servicebus").ServiceBusManagementClient;
+
+const dotenv = require("dotenv");
+dotenv.config();
+
+const defaultLockDuration = "PT30S"; // 30 seconds in ISO 8601 FORMAT - equivalent to "P0Y0M0DT0H0M30S"
+
+EnvVarKeys = {
+  SERVICEBUS_CONNECTION_STRING: "SERVICEBUS_CONNECTION_STRING",
+  QUEUE_NAME: "QUEUE_NAME",
+  QUEUE_NAME_NO_PARTITION: "QUEUE_NAME_NO_PARTITION",
+  QUEUE_NAME_SESSION: "QUEUE_NAME_SESSION",
+  QUEUE_NAME_NO_PARTITION_SESSION: "QUEUE_NAME_NO_PARTITION_SESSION",
+  TOPIC_NAME: "TOPIC_NAME",
+  TOPIC_NAME_NO_PARTITION: "TOPIC_NAME_NO_PARTITION",
+  TOPIC_NAME_SESSION: "TOPIC_NAME_SESSION",
+  TOPIC_NAME_NO_PARTITION_SESSION: "TOPIC_NAME_NO_PARTITION_SESSION",
+  SUBSCRIPTION_NAME: "SUBSCRIPTION_NAME",
+  SUBSCRIPTION_NAME_NO_PARTITION: "SUBSCRIPTION_NAME_NO_PARTITION",
+  SUBSCRIPTION_NAME_SESSION: "SUBSCRIPTION_NAME_SESSION",
+  SUBSCRIPTION_NAME_NO_PARTITION_SESSION: "SUBSCRIPTION_NAME_NO_PARTITION_SESSION",
+  TOPIC_FILTER_NAME: "TOPIC_FILTER_NAME",
+  TOPIC_FILTER_SUBSCRIPTION_NAME: "TOPIC_FILTER_SUBSCRIPTION_NAME",
+  TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME: "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
```

Comment:
Shouldnt all of the above except SERVICe_BUS_CONNECTION_STRING have the _BROWSER suffix?

Reply:
Nope, not here..